psw/urts/enclave.cpp

```diff
@@ -43,7 +43,7 @@
 using namespace std;
 
 int do_ecall(const int fn, const void *ocall_table, const void *ms, CTrustThread *trust_thread);
-int do_ocall(const bridge_fn_t bridge, void *ms);
+int do_ocall(const bridge_fn_t bridge, sgx_enclave_id_t enclave_id, void *ms);
```

Comment:
So this seems not backward compatible, because the new uRTS cannot do ocall for old Enclaves.  A backward compatible way might be uRTS knows the difference of these 2 types of ocalls and switch the ocall API during runtime.